.travis.yml

```diff
@@ -0,0 +1,14 @@
+version: ~> 1.0 # needed for imports
+
+import: scala/scala-dev:travis/default.yml
+
+language: scala
+
+scala: 2.12.10
+
+env:
+  global:
```

Comment:
what does adding `global:` do? I usually just see `env:` by itself. just curious

Reply:
This specimen of `global:` does a good job at looking important whereas in reality it's merely a pasted copy. Not sure what else it does all day.